draft-wood-cfrg-eddsa-blinding.md

```diff
@@ -87,28 +113,30 @@ blinding.
 The following terms are used throughout this document to describe the blinding modification.
 
 - `G`: The standard base point.
-- `sk`: An EdDSA private key, which is a randomly generated private 
-  seed of length 32 bytes or 57 bytes according to {{RFC8032, Section 5.1.5}}
-  or {{RFC8032, Section 5.2.5}}, respectively.
+- `sk`: A signature scheme private key. For EdDSA, this is a a randomly generated 
+  private seed of length 32 bytes or 57 bytes according to {{RFC8032, Section 5.1.5}}
+  or {{RFC8032, Section 5.2.5}}, respectively. For {{ECDSA}}, `sk` is a random scalar
+  in the prime-order elliptic curve group.
 - `pk(sk)`: The public key corresponding to the private key `sk`.
 - `concat(x0, ..., xN)`: Concatenation of byte strings.
   `concat(0x01, 0x0203, 0x040506) = 0x010203040506`.
-- ScalarMult(pk, k): Multiply the EdDSA public key pk by scalar k, producing a new
+- ScalarMult(pk, k): Multiply the public key pk by scalar k, producing a new
   public key as a result.
+- ModInverse(x, L): Compute the multiplicative inverse of x modulo L.
 
-In pseudocode descriptions below, integer multiplication of two EdDSA scalar values
-is denoted by the \* operator. For example, the product of two scalars `x` and `y`
-is denoted as `x * y`.
+In pseudocode descriptions below, integer multiplication of two scalar values is denoted 
+by the \* operator. For example, the product of two scalars `x` and `y` is denoted as `x * y`.
 
 # Key Blinding
 
 At a high level, key blinding allows signers to use private keys to blind their signing
 key such that any signature produced under the blinded key pair is unlinkable to the
-original signing key pair. Similar to the EdDSA private key used for signing, the blind 
-is also an EdDSA private key. That is, the blind is a 32-byte or 57-byte random seed for
-Ed25519 or Ed448 variants, respectively.
+original signing key pair. Similar to the private key used for signing, the blind is 
+also a private key. For example, the blind is a 32-byte or 57-byte random seed for 
```

Comment:
Maybe "the blind must also remain secret"?

Calling two different things "private key" may be a little big confusing, especially since the blind doesn't have a public counterpart.

Reply:
The point I was trying to convey is that the blind is a secret generated just like any other private signing key. Let me try to reformulate somewhat differently.